README.md

```diff
@@ -19,67 +19,21 @@ Docs at [https://dimagi.github.io/commcare-cloud/](https://dimagi.github.io/comm
 
 
 # Install and setup
-You will need python 2.7 and `virtualenvwrapper` installed
-to follow these instructions.
-
-Clone the commcare-cloud repo
-(suggested location alongside commcare-hq, if you have that repo cloned as well):
-
-```
-git clone https://github.com/dimagi/commcare-cloud.git
-cd commcare-cloud
-```
-
-Now make a virtualenv for ansible:
-
-```
-mkvirtualenv ansible
-```
-
-If you want `workon ansible` to always bring you to this directory, then you can also run
-
-```
-setvirtualenvproject
-```
-
-at this time.
-
-**Note**: If you already have commcare-cloud cloned, then just enter that directory
-and update it with
+You will need python 2.7 and `virtualenvwrapper` installed to follow these instructions:
 
 ```
-git pull
+sudo apt-get install git python-dev python-pip
```

Comment:
This probably only works on debian-ish linux distros. Is it worth mentioning that?